sdk/eventhub/azure-eventhub/azure/eventhub/_common.py

```diff
@@ -541,18 +591,27 @@ def _load_events(self, events):
                     "the Event Hub frame size limit. Please send a smaller collection of EventData "
                     "or use EventDataBatch, which is guaranteed to be under the frame size limit"
                 )
+    
+    @property
+    def message(self) -> Union["uamqp_BatchMessage", LegacyBatchMessage]:
```

Comment:
In EventData, I'm always returning LegacyMessage, b/c we don't store uamqp Messages in EventData. However, here, since we do store the uamqp.BatchMessage in self._message, I'm also returning that.

Will double check that message getters/setters are not showing up in docs/apiview.

Reply:
as per Johan + Anna: adding deprecation warnings to this instead.